common/components/controllers/AboutUserController.jsx

```diff
@@ -0,0 +1,156 @@
+// @flow
+
+import React from 'react';
+import TagsDisplay from '../common/tags/TagsDisplay.jsx'
+import {LinkNames} from "../constants/LinkConstants.js";
+import {FileCategoryNames} from "../constants/FileConstants.js";
+import {UserAPIData} from "../utils/UserAPIUtils.js";
+import UserAPIUtils from "../utils/UserAPIUtils.js";
+import {FileInfo} from "../common/FileInfo.jsx";
+import _ from 'lodash'
+
+type State = {|
+  user: ?UserAPIData,
+|};
+
+class AboutUserController extends React.PureComponent<{||}, State> {
+  constructor(): void {
+    super();
+
+    this.state = {
+      user: null
+    };
+  }
+
+  componentDidMount() {
+    var userId = (new RegExp("id=([^&]+)")).exec(document.location.search)[1];
+    UserAPIUtils.fetchUserDetails(userId, this.loadUserDetails.bind(this));
+  }
+
+  loadUserDetails(user: UserAPIData) {
+    this.setState({
+      user: user
+    });
+  }
+
+  render(): React$Node {
+    return this.state.user ? this._renderDetails() : <div>Loading...</div>
+  }
+
+  _renderDetails(): React$Node {
+    const user: UserAPIData = this.state.user;
+    return (
+      <div className="AboutProjectController-root">
+        <div className="container-fluid">
+          <div style={{backgroundColor: "white"}}>
+            <div className="row" style={{margin: "30px 0 0 0", padding: "10px 0"}}>
```

Comment:
Do you have an opinion on using inline styles like this?  In this instance, I actually like it, since this kind of box styling is really local to the component, and creating a class seems like an unnecessary abstraction.

Reply:
We do it both ways in the code, and generally leave it up to the authors to decide which way makes sense.